docs/sources/enterprise/access-control/manage-rbac-roles.md

```diff
@@ -277,46 +246,102 @@ curl --location --request POST '<grafana_url>/api/access-control/roles/' \
 
 1. Reload the provisioning configuration file.
 
    For more information about reloading the provisioning configuration at runtime, refer to [Reload provisioning configurations]({{< relref "../../http_api/admin/#reload-provisioning-configurations" >}}).
 
-The following example removes the `fixed:permissions:admin` from the `Grafana Admin` basic role.
+The following example modifies the `Grafana Admin` basic role permissions.
+It removes from it, the permission to list, grant and revoke roles to teams.
+It also adds to it the permission to read and write grafana folders.
 
 ```yaml
 # config file version
-apiVersion: 1
+apiVersion: 2
 
-# list of default basic role assignments that should be removed
-removeDefaultAssignments:
-  - builtInRole: 'Grafana Admin'
-    fixedRole: 'fixed:permissions:admin'
+roles:
+  - name: 'basic:grafana_admin'
+    global: true
+    version: 3
+    from:
+      - name: 'basic:grafana_admin'
+        global: true
+    permissions:
+      # Permissions to remove
+      - action: 'teams.roles:list'
+        scope: 'teams:*'
+        state: 'absent'
+      - action: 'teams.roles:remove'
+        scope: 'permissions:type:delegate'
+        state: 'absent'
+      - action: 'teams.roles:add'
+        scope: 'permissions:type:delegate'
+        state: 'absent'
+      # Permissions to add
+      - action: 'folders:read'
+        scope: 'folder:*'
+      - action: 'folders:write'
+        scope: 'folder:*'
 ```
 
-You can also remove fixed roles from basic roles using the API. Refer to the [RBAC HTTP API]({{< relref "../../http_api/access_control.md#remove-a-built-in-role-assignment" >}}) for more details.
+> **Note**: You can add multiple `fixed`, `basic` or `custom` roles to the `from` section. Their permissions will be copied and added to the basic role.
+> <br/> > **Note**: Make sure to **increment** the role version for the changes to be accounted for.
+
+You can also change basic roles' permissions using the API. Refer to the [RBAC HTTP API]({{< relref "../../http_api/access_control.md#update-a-role" >}}) for more details.
+
+## Reset basic roles permissions to factory
+
+This section describes how to reset the basic roles permissions to factory defaults:
+
+1. Open the YAML configuration file and locate the `roles` section.
+
+1. Grant the `action: "roles:write", scope: "permissions:type:escalate` permission to `Grafana Admin`.
+
+   ```yaml
+   apiVersion: 2
+   roles:
+     - name: 'basic:grafana_admin'
+       global: true
+       version: 3
+       from:
+         - name: 'basic:grafana_admin'
+           global: true
+       permissions:
+         # Permission allowing to reset basic roles to factory
+         - action: 'roles:write'
+          scope: 'permissions:type:escalate'
+   ```
+
+   **Note**: This permission has not been granted to any basic roles by default, since it could allow in some cases, privilege escalations.
```

Comment:
```suggestion
> **Note**: This permission has not been granted to any basic roles by default, because in some cases it does not allow privilege escalations.
```

Reply:
I think this changes what I'm trying to express.Would this be better:

```suggestion
   **Note**: This permission has not been granted to any basic roles by default, since users could acquire more permissions than they previously had through the basic role permission reset.
```